core/templates/pages/learner-dashboard-page/card-display/card-display.component.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component to display lesson cards based on tab
+ */
+import {Component, ElementRef, Input, ViewChild} from '@angular/core';
+import {downgradeComponent} from '@angular/upgrade/static';
+
+@Component({
+  selector: 'card-display',
```

Comment:
Can we make the selector name `oppia-card-display` to make the component consistent throughout the codebase?

Reply:
Done